data/src/main/java/utill/TokenAuthenticator.kt

```
@@ -0,0 +1,54 @@
+            val response = runBlocking { authApi.gauthAccess() }
+            runBlocking {
+                with(localDataSource){
+                    setAccessToken(response.accessToken)
+                    setAccessTime(response.accessTokenExpiresIn)
+                    setRefreshToken(response.refreshToken)
+                    setRefreshTime(response.refreshTokenExpiresIn)
+                }
+
+            }
+            response.accessToken
+        }catch (e: Exception){
```

detekt

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)